src/remote-reply/index.js

```diff
@@ -0,0 +1,16 @@
+import { render } from '@wordpress/element';
+import domReady from '@wordpress/dom-ready';
+import RemoteReply from './remote-reply';
+
+let id = 1;
+function getUniqueId() {
+	return `activitypub-remote-reply-link-${ id++ }`;
+}
+
+domReady( () => {
+	// iterate over a nodelist
+	[].forEach.call( document.querySelectorAll( '.activitypub-remote-reply' ), ( element ) => {
+		const attrs = JSON.parse( element.dataset.attrs );
+		render( <RemoteReply { ...attrs } id={ getUniqueId() } useId={ true } />, element );
```

Comment:
React has recently deprecated the `React.render` method in favour of `createRoot`. It would now be something like 

```js
import { createRoot } from '@wordpress/element`;

// line instead of `render( <RemoteReply //etc`
createRoot( element).render( <RemoteReply { ...attrs } id={ getUniqueId() } useId={ true } )
```

(I need to go update other uses too)